run_modal_docker.py

```
@@ -0,0 +1,244 @@
+    import os
+    
+    # Hugging Faceにログイン
+    try:
+        from huggingface_hub import login
+        hf_token = os.environ.get("HF_TOKEN")
+        if hf_token:
+            login(token=hf_token)
+            print("✅ Successfully logged in to Hugging Face")
+        else:
+            print("⚠️ No HF_TOKEN found")
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)